### DISCUSSION TITLE
Why does the global style sheet not support the src folder?

### DISCUSSION THREAD

**Discussion by zao95** [vercel/next.js#21218] [Help]
I understand that the pages folder can be placed in src.

However, when using the global stylesheet, it does not apply to root/src/pages/_app.js and forces the root/pages/_app.js.

Why doesn't the global stylesheet also support putting it in the root/src/pages/_app.js?

If global stylesheet is too old to support properly, please let me know which method to use.

**Comment by vassbence** [comment]
Next definitely supports this, could you share some code or reproduction please?

**Reply by zao95** [reply]
I misunderstood the official document that should be included in pages/_app.js.
It work!
Thank you for your reply.
